ksqldb-engine/src/main/java/io/confluent/ksql/analyzer/PullQueryValidator.java

```diff
@@ -91,29 +89,47 @@ public void validate(final Analysis analysis) {
     }
   }
 
-  private static boolean disallowedColumnNameInSelectClause(final Analysis analysis) {
-    return analysis.getSelectItems()
+  private static Optional<String> disallowedColumnNameInSelectClause(final Analysis analysis) {
+    final Optional<ColumnName> disallowed =  analysis.getSelectItems()
         .stream()
         .filter(col -> col instanceof SingleColumn) //filter out select *
         .map(SingleColumn.class::cast)
         .map(SingleColumn::getExpression)
         .map(ColumnExtractor::extractColumns)
         .flatMap(Collection::stream)
         .map(ColumnReferenceExp::getColumnName)
-        .anyMatch(name -> disallowedInPullQueries(name, analysis.getKsqlConfig()));
+        .filter(name -> disallowedInPullQueries(name, analysis.getKsqlConfig()))
+        .findFirst();
```

Comment:
Instead of returning just the first unsupported column, it'd be more helpful to the user to return all unsupported columns.